# My Jupyter Notebook on IBM Watson Studio

**Leanna**

Data Scientist

*I am interested in data science because I love to work with data and use data to tell stories!*

### In the next cell I will show

+ how to assign values to variables
+ how to do arithmetic with varables
+ print the results

In [2]:
x=100
y=3
z=(x*y)+y
print("the value of Z is-->",z)

the value of Z is--> 303


# Bulleted List
+ a
+ b
+ c

# Numbered List
1. x
1. y
1. z

<h1 style="text-align: center;">Table center aligned</h1>

|Variables|
|:-------:|
|x=100|
|y=3|
|z=(x*y)+y=303|